docusaurus/docs/shared/plugin-anatomy.md

```diff
@@ -1,25 +1,23 @@
-Plugins come in different shapes and sizes. Before we dive deeper, let's look at some of the properties that are shared by all of them.
-
-Every plugin you create will require at least two files: `plugin.json` and `src/module.ts`.
+Plugins come in different shapes and sizes, however every plugin you create will require at least two files: `plugin.json` and `src/module.ts`.
 
 ### plugin.json
```

Comment:
```suggestion
### `plugin.json`
```

Reply:
happy to make this change, but shouldnt we also make it for module.ts?